pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.beeswax</groupId>
   <artifactId>beeswax-api</artifactId>
-  <version>2021-02-05</version>
+  <version>2023-08-25</version>
```

Comment:
Is this change only for Display purpose or does it trigger any build/package pipelines?

Reply:
It's part of the process outlined in [this wiki](https://wiki.freewheel.tv/display/BWEN/How+To+update+the+public+Beeswax+API+repo). We've previously made some changes in this repo but haven't actually released it for a few years.

Reply:
To answer your question - no it doesn't trigger any pipelines but I'll use it to deploy to maven.